app/views/donation_sites/show.html.erb

```diff
@@ -31,16 +31,22 @@
           <div class="card-body p-0">
             <table class="table">
               <thead>
-              <tr>
-                <th><h3>Donation Site Name</h3></th>
-                <th><h3>Donation Site Address</h3></th>
-              </tr>
+                <tr>
+                  <th<h3>Donation Site Name</h3></th>
+                  <th><h3>Donation Site Address</h3></th>
+                  <th><h3>Contact Name</h3></th>
+                  <th><h3>Email</h3></th>
+                  <th><h3>Phone</h3></th>
+                </tr>
               </thead>
               <tbody>
-              <tr>
-                <td><h4><%= @donation_site.name %></h4></td>
-                <td><h4><%= @donation_site.address %></h4></td>
-              </tr>
+                <tr>
+                  <td><%= @donation_site.name %></td>
+                  <td><%= @donation_site.address %></td>
+                  <td><%= @donation_site.contact_name || "N/A" %></td>
```

Comment:
You have a default setup for `contact_name` but no default to be displayed for email/phone. Any motivation for that? If not, might be best to move to make it consistent